hal/src/main/native/include/hal/Errors.h

```diff
@@ -120,6 +120,10 @@
 #define HAL_SIM_NOT_SUPPORTED -1155
 #define HAL_SIM_NOT_SUPPORTED_MESSAGE "HAL: Method not supported in sim"
 
+#define HAL_CAN_BUFFER_OVERRUN -35007
```

Comment:
What's the source for this value?  It's not adjacant to the other values...

Reply:
There isn't one. It's not defined in either the emb can header or the netcomm header. And I find 1 obscure Google reference to it. But that its what it is